src/app/actions/actions.ts

```diff
@@ -0,0 +1,15 @@
+"use server";
+import { PrismaClient } from "@prisma/client";
+
+const prisma = new PrismaClient();
```

Comment:
we will need to change it later to using prisma instance from db file that @mproznydev created in his mr

Reply:
Yup, I will add TODO and change it